src/app/Models/Traits/CrudTrait.php

```diff
@@ -296,4 +316,62 @@ public function translationEnabled()
 
         return false;
     }
+
```

Comment:
Agreed. Splited into a new trait to have this logic separated. Not sure i agree 100% with naming, I would call it `ModelIdentifiableAttribute` because, well, we don't check if it `Has`, we allways return `Model` identifiableAttribute!

Reply:
Hmm... indeed. It's good that we have the luxury to think about better names. I'll add a new comment in the latest batch of updates to talk about this - could be come a longer conversation.